Util.py

```diff
@@ -2975,6 +2975,22 @@ def get_utc_start_time_filename_part():
   return time.strftime("%Y-%m-%d-%H-%M-%S", time.gmtime(start_time))
 
 
+def maybe_make_dirs(dirname):
+  """
+  Creates the directory if it does not yet exist.
+  :param dirname: The absolute path of the directory
```

Comment:
There should be a type in the docstring (`:param str dirname:`).
Also better make a newline in the docstring after the description, before the params.